content/docs/services/elasticsearch17.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: Elasticsearch 1.7
 name: "elasticsearch17"
-description: "[Elasticsearch](https://www.elastic.co/) version 1.7"
+description: "[Elasticsearch](https://www.elastic.co/products/elasticsearch) version 1.7 a distributed, RESTful search and analytics engine"
```

Comment:
This needs something like a colon after the version number, and I suspect links aren't a good idea since they wouldn't show up properly in the CLI - `Elasticsearch version 2.3: a distributed, RESTful search and analytics engine`